app.js

```diff
@@ -54,6 +54,20 @@ if (argv.load) {
     isPersistence = true;
 }
 
+if(argv.accounts) { 
+    LOG_APPJS(`Bootstrapping from account fixture files: ${argv.accounts}`);
+    let accountsPath = argv.accounts;
+    if(!fs.existsSync(accountsPath)) {
+        throw new Error('Account Path Invalid');
```

Comment:
If you throw here, the process will halt, right? I think we should fall through to the default 10 accounts if the user accidentally calls the CLI wrongly.